lib/Doctrine/Common/Collections/ExpressionBuilder.php

```diff
@@ -17,7 +17,7 @@
 class ExpressionBuilder
 {
     /**
-     * @param mixed $x
+     * @param mixed ...$x
```

Comment:
This syntax is not really understood by current tooling 🤔

Reply:
https://github.com/vimeo/psalm/blob/master/tests/VariadicTest.php

Reply:
Oh, that's very good news!